docs/development/README.md

```diff
@@ -17,4 +17,5 @@ Here are some tips to help during development.
 - [mxgraph version bump](./mxgraph-version-bump.md)
 
 ## Misc
+- [documentation](./documentation.md)
```

Comment:
👍

Reply:
I prefer to rename this page.
All the .adoc & .md page are documentation. It's a little confusing.

If it's the doc for the developpers, I'd rather call it **CONTRIBUTING CODE** (or somthing like thath), like https://p5js.org/contributor-docs/#/

Reply:
We are going to create a new PR just once this once is merged to clarify the user and development documentation to avoid mixup.
All asciidoctor files will be store in a new folder (name to be defined: user, user-guide, usage, ....) at the root of the docs folder.

About the currently named `documentation.md`: it is explaining how we document the project, so we can rename it into `how-to-document` or something else. But it is not related to how to contribute.

Reply:
Thanks for the explanation 😉

Reply:
Decisions for the names of the new doc folders
- contributors: for anything about development and contribution
- users: for the doc files that produce the html docs which target uses/integration of the lib

Reply:
see #904